python/sglang/srt/layers/moe/fused_moe_triton/layer.py

```diff
@@ -205,19 +221,46 @@ def forward_cpu(
         correction_bias: Optional[torch.Tensor] = None,
         activation: str = "silu",
     ) -> torch.Tensor:
-        return moe_forward_native(
-            layer,
-            x,
-            use_grouped_topk,
-            top_k,
-            router_logits,
-            renormalize,
-            topk_group,
-            num_expert_group,
-            custom_routing_function,
-            correction_bias,
-            activation,
-        )
+        assert activation == "silu", f"activation = {activation} is not supported."
+
+        if cpu_has_amx_support():
+            from sgl_kernel.cpu import fused_experts
```

Comment:
I remember that import from function should be able to be cached by python, so that the `import` does not happen each time the function is called. Could you please confirm?

Reply:
I refactored the code in https://github.com/mingfeima/sglang/pull/8/commits/5404d16098d943ff40a351e821b341c226abe708 to run `import sgl_kernel.cpu` once at the beginning of the file if `cpu_has_amx_support()`. Does this look good?